frame/evm/precompile/dispatch/Cargo.toml

```diff
@@ -12,7 +12,7 @@ description = "DISPATCH precompiles for EVM pallet."
 sp-core = { version = "3.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
 sp-io = { version = "3.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
 frame-support = { version = "3.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "frontier" }
-pallet-evm = { version = "3.0.0", default-features = false, path = "../.." }
+pallet-evm = { version = "3.0.1-dev", default-features = false, path = "../.." }
```

Comment:
Any bump of dependency should be recursive. See https://github.com/paritytech/frontier#versioning

Alternatively, since it's a patch and no interface is changing, you can also just revert those dependency bumps.

Reply:
Ok, I reverted the dependency bumps. There are no interface changes so I think this is appropriate.